KomaMRIBase/src/datatypes/phantom/motion/SimpleMotion.jl

```diff
@@ -96,8 +96,8 @@ function times(motion::SimpleMotion)
     return nodes
 end
 
-function sort_motions!(motion::SimpleMotion)
-    return sort!(motion.types; by=m -> times(m)[1])
+function initialize_motion!(motion::SimpleMotion)
+    sort!(motion.types; by=m -> times(m)[1])
```

Comment:
Don't delete the `return` it is required by the Julia blue code style.

Reply:
Solved in [47ccfc0](https://github.com/JuliaHealth/KomaMRI.jl/pull/408/commits/47ccfc0ee67c813c5da8b5d23129ef1ef39568b4)